examples/Demo/Shared/Pages/Persona/Examples/PersonaDismiss.razor

```diff
@@ -0,0 +1,20 @@
+﻿<div hidden="@this.Hidden">
+    <FluentPersona Name="Kevin Sturgis"
+                   Status="PresenceStatus.Available"
+                   StatusSize="PresenceBadgeSize.Small"
+                   StatusTitle="He is available"
+                   Image="https://res-1.cdn.office.net/files/fabric-cdn-prod_20230815.002/office-ui-fabric-react-assets/persona-male.png"
```

Comment:
Same... Could you use the `@DataSource.SamplePicture` image? To be independent of external resources (which can be removed later)

Reply:
I fixed both remarks.